plugins/com.google.cloud.tools.eclipse.welcome/plugin.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+   <extension point="org.eclipse.ui.intro.configExtension">
```

Comment:
I may be wrong, but I think this extension is about the main intro about the product, something you will create when you create a new RCP application. In our IDE case, I think the Eclipse IDE main intro is already there. We are adding a link to this intro (in a top-level page) by extending it, so it seems different.